app/models/group.rb

```diff
@@ -1,4 +1,6 @@
 class Group < ApplicationRecord
   has_many :messages
   has_one_attached :photo
+  has_many :memberships
```

Comment:
I don't think you need that method. Will you ever use
Group.memberships ?

Reply:
We need to have it to have Group

Reply:
For the group chat group.memberships would allow us to check who is allowed and display the list of group members. It makes sense for.me.to.keep it.

Reply:
Would leave it like this